azure-pipelines.yml

```diff
@@ -6,13 +6,22 @@
 trigger:
   - master
 
+resources:
+  containers:
+    - container: wpilib2019
+      image: wpilib/roborio-cross-ubuntu:2019-18.04
+
 jobs:
   - job: Linux
     pool:
       vmImage: 'Ubuntu 16.04'
```

Comment:
I see that this was changed to 18.04 in commit 5e8ee9d but changed back to 16.04 in 63797a9  Since other changes also happened in 63797a9 it wasn't clear to me if this reset was intentional.

Reply:
It was. Azure pipelines only has Ubuntu 16.04, not 18.04